ArcadiaParties.API/Controllers/CalendarController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using ArcadiaParties.CQRS.Queries;
+using ArcadiaParties.Data.Abstractions.DTOs;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace ArcadiaParties.API.Controllers
+{
+    [Route("api/[controller]")]
+    public class CalendarController : ControllerBase
+    {
+        private readonly IMediator _mediator;
+
+        public CalendarController(IMediator mediator)
+        {
+            _mediator = mediator;
+        }
+
+        [ProducesResponseType(typeof(IEnumerable<UserForCalendarDTO>), StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [SwaggerOperation(
+             Summary = "Returns users of department"
+        )]
+        [Authorize]
+        [HttpGet("{id}")]
+        public async Task<IActionResult> GetUsersOfDepartment(int id, CancellationToken cancellationToken)
+        {
+            var query = new GetUsersOfDepartmentQuery(id);
```

Comment:
And what if there is not department with specified id?

Reply:
Department Exists added